src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Bit.Client.Web.BlazorUI.Playground.Web.csproj

```diff
@@ -34,4 +34,11 @@
         <ProjectReference Include="..\Shared\Bit.Client.Web.BlazorUI.Playground.Shared.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Content Update="wwwroot\fonts\fabric-icons.woff">
+        <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+        <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      </Content>
+    </ItemGroup>
+
```

Comment:
I think these changes must not be committed!

Reply:
@sajadkardel Do we really need this? @sajadkardel

Reply:
no, these changes are redundant

Reply:
No, I was wrong! I'm fixing it right now :) @ysmoradi